src/lang/std/artifactGraph.ts

```diff
@@ -393,6 +393,32 @@ export function getArtifactsToUpdate({
         artifact: { ...path, sweepId: id },
       })
     return returnArr
+  } else if (
+    cmd.type === 'loft' &&
+    response.type === 'modeling' &&
+    response.data.modeling_response.type === 'loft'
+  ) {
+    returnArr.push({
+      id,
+      artifact: {
+        type: 'sweep',
+        subType: 'loft',
+        id,
+        pathId: response.data.modeling_response.data.solid_id,
```

Comment:
No idea what we should do here. Figured storing this magic solid_id specific to lofts (vs extrude or revolve) could be useful. This may or may not be the cause of the kcl error I'm seeing on this branch